plugins/jobtype/src/azkaban/jobtype/HadoopPigJob.java

```diff
@@ -85,11 +85,6 @@ public HadoopPigJob(String jobid, Props sysProps, Props jobProps, Logger log)
     obtainTokens = getSysProps().getBoolean("obtain.binary.token", false);
     userPigJar = getJobProps().getBoolean("use.user.pig.jar", false);
 
-    // Fetch HCatalog token if enabled
-    boolean obtainHcatToken = getSysProps().getBoolean(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, false);
-    obtainHcatToken = getJobProps().getBoolean(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, obtainHcatToken);
-    getSysProps().put(HadoopSecurityManager.OBTAIN_HCAT_TOKEN, Boolean.toString(obtainHcatToken));
-
     if (shouldProxy) {
       getLog().info("Initiating hadoop security manager.");
       try {
```

Comment:
I think we should also revert getSysProps() -> sysProps (line 91) as we are not modifying properties, so it should be good to stick which actual parameter of the method ?